dm/dm-webui-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 使用 WebUI 管理 DM 迁移任务
+summary: 学习如何使用 WebUI 来方便的管理数据迁移任务
+---
+
+# 使用 WebUI 管理 DM 迁移任务
+
+DM WebUI 是为了方便大量迁移任务的管理、简化操作步骤的目的开发，其主要特性包含运行状态监控，上游数据源配置，以及迁移任务管理等。DM WebUI 可以从任意 master 节点访问，其端口与 OpenAPI 保持一致，默认为 8261。访问地址示例：`http://{master_ip}:{master_port}/dashboard/`。
+
+> **注意：**
+>
+> - DM WebUI 当前为实验特性，尚不建议用于生产环境。
+> 
+> - DM WebUI 中 task 的生命周期有所改变，不建议与 dmctl 同时使用。
+
+## 实验特性开关
+
+若要使用 DM WebUI，需要在 master 配置中开启`openapi`配置：
+
+```
+openapi = true
+```
+
+## Dashboard
+
+Dashboard 是内嵌了 DM 的 Grafana Dashboard，包含两个视图`Standard`和`Professional`分别展示了不同角度的监控信息。
+
+如果要在 DM WebUI 中正确显示 Dashboard，首次部署必须确认以下两个步骤正确完成：
+
+1. 部署时正确安装监控相关组件：`monitoring_servers`和`grafana_servers` ，其中`grafana_servers`的配置必须如下所示：
+
+    ```
+    grafana_servers:
+      - host: 10.0.1.14
+        # port: 3000
+        # deploy_dir: /tidb-deploy/grafana-3000
+        config:
+          auth.anonymous.enabled: true
+          security.allow_embedding: true
+    ```
+
+2. 若`grafana_servers`使用了非默认的 IP 和端口，则需要在 WebUI 的 Dashboard 界面填写正确的 IP 和端口。
+
+若 DM 是从旧版本升级，则需要手动修改 Grafana 的配置，首先编辑`/{deploy-dir}/grafana-{port}/conf/grafana.ini`，修改以下两项配置：
+
+```ini
+[auth.anonymous]
+enabled = true
+
+[security]
+allow_embedding = true
+```
+
+然后执行`tiup dm reload`使新的配置生效即可。
+
+## 任务列表
+
+任务列表用于管理所有的迁移任务。与 v6.0 以下版本不同，WebUI 首先使用了新的生命周期管理（dmctl 预计将在 v6.1 版本同步修改）。这将导致操作方式与之前版本有所不同，可能需要一些适应。
```

Comment:
It is recommended that this point is not included in the documentation, as the work is not yet fully completed